app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">Vocable</string>
+  <string name="select_something">Hanxinl75@gmail. com</string>
+  <string name="phrases_page_number">hanxinl75@gmail.com</string>
```

Comment:
I know this just came from Crowdin but this doesn't seem right lol

Reply:
Oh that _is_ weird, I hadn't actually noticed it before...I'll make sure it gets deleted. Actually, that'll make for a nice test of the CrowdIn integration